grid.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+import Ogre
+
+class GridFloor:
+    def __init__(self, scene, material_name):
+        self.scn_mgr = scene
+        self.material_name = material_name
+        self.drawQueue = 1
+        self.plane_names = ["YZPlane", "XZPlane", "XYPlane"]
+        self.normals = [
+            (1, 0, 0),  # YZPlane normal
+            (0, 1, 0),  # XZPlane normal
+            (0, 0, 1)   # XYPlane normal
+            ]
+        self.axis_colors = [
+            Ogre.ColourValue(1, 0, 0, 1),
+            Ogre.ColourValue(0, 1, 0, 1),
+            Ogre.ColourValue(0, 0, 1, 1)
+            ]
+        self.min_val = -5
+        self.max_val = 5
+
+    def show_plane(self, plane):
+        for p in range(0, 3):
+            plane_name = "MeshViewer/" + self.plane_names[p]
+            plane_node = self.scn_mgr.getSceneNode(plane_name)
+            plane_node.setVisible(p == plane)
+
+    def create_planes(self):
+        for plane in range(0, 3):
+            plane_name = "MeshViewer/" + self.plane_names[plane]
+            self.create_plane(plane, self.min_val, self.max_val)
+
+    def create_plane(self, plane, min_val, max_val):
+        cl = None
+        axis_color = None
+        plane_name = "MeshViewer/" + self.plane_names[plane]
+
+        normal = self.normals[plane]
+
+        color_X = self.axis_colors[0]
+        color_Y = self.axis_colors[1]
+        color_Z = self.axis_colors[2]
+
+        if plane == 0:
+            axis_color = [color_Y, color_Z]
```

Comment:
```suggestion
            axis_color = [self.axis_colors[(plane+1)%3], self.axis_colors[(plane+2)%3]]
```

Reply:
Cool, but I would like to leave it in a more undestandable way.

I know it is more efficient but ist more readable for me 😅